_data/locales/en.yml

```diff
@@ -5,12 +5,12 @@ en:
     question: What Does Homebrew Do?
     what: Homebrew installs <a href="https://formulae.brew.sh/formula/" title="List of Homebrew packages">the stuff you need</a> that Apple (or your Linux system) didn’t.
     how: Homebrew installs packages to their own directory and then symlinks their files into <code>/usr/local</code>.
-    prefix: Homebrew won’t install files outside its prefix, and you can place a Homebrew installation wherever you like.
+    prefix: Homebrew won’t install files outside its prefix and you can place a Homebrew installation wherever you like.
     createpackages: Trivially create your own Homebrew packages.
     hack: It's all Git and Ruby underneath, so hack away with the knowledge that you can easily revert your modifications and merge upstream updates.
     formula: "Homebrew formulae are simple Ruby scripts:"
     editor: opens in $EDITOR!
-    complement: Homebrew complements macOS (or your Linux system). Install your RubyGems with <code>gem</code>, and their dependencies with <code>brew</code>.
+    complement: Homebrew complements macOS (or your Linux system). Install your RubyGems with <code>gem</code> and their dependencies with <code>brew</code>.
```

Comment:
Are we not diminishing Linux by putting it in parentheses ?

Reply:
This comma *was* in the wrong place. It should be moved to line 8. I'm glad that you were able to find a spare comma, one that you are now able to employ elsewhere. I didn't really want to lend you any of my commas. I'm attached to them. They have sentimental value.

Reply:
@javian yes, we are a bit but that’s because it’s implying WSL too and Homebrew will always be a primarily macOS package manager. It also implies the “missing” is for macOS but not for Linux.